libs/editor/WordPressEditor/src/main/res/values/strings.xml

```diff
@@ -94,4 +94,18 @@
     <string name="editor_dropped_title_images_not_allowed">Dropping images in the Title is not allowed</string>
     <string name="editor_dropped_html_images_not_allowed">Dropping images while in HTML mode is not allowed</string>
     <string name="editor_dropped_unsupported_files">Warning: not all dropped items are supported!</string>
+
+    <!-- AZTEC -->
+    <string name="edit_hint">Share your story here&#8230;</string>
+    <string name="source_hint">&lt;p&gt;Share your story here&#8230;&lt;/p&gt;</string>
```

Comment:
Not blocking for merging this but I think we should add the `<p>` and `</p>` programmatically (and reuse `edit_hint`), I fear translators will make mistakes on this string.